metagpt/provider/litellm.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/08/04 10:00
+@Author  : ishaan-jaff
+@File    : openai.py
+"""
+from metagpt.provider.base_gpt_api import BaseGPTAPI
+from metagpt.provider.openai_api import CostManager, RateLimiter
+from metagpt.config import CONFIG
+
+class liteLLM(BaseGPTAPI, RateLimiter):
+    def __init__(self):
+        self.__init_openai(CONFIG)
+        self.model = CONFIG.model # litellm model
+        self._cost_manager = CostManager()
+        RateLimiter.__init__(self, rpm=self.rpm)
+
+    def _chat_completion(self, messages: list[dict], model: str) -> dict:
```

Comment:
It is necessary to implement the 'acompletion_text' function in the liteLLM class because it is being used within the Action.